_security/access-control/api.md

```diff
@@ -940,6 +940,22 @@ PUT _plugins/_security/api/rolesmapping/<role>
 ```
 {% include copy-curl.html %}
 
+#### Host-based role mapping
+
+The `hosts` parameter maps requests originating from specific IP addresses or hostnames to the given role. This is useful when you want to assign roles based on the client's source address:
```

Comment:
Need to check if CIDR blocks are also allowed, but otherwise the content of this PR looks good to me.